_posts/2015-09-04-channel.html

```diff
@@ -2955,6 +2955,234 @@
 	...
   }</code></pre>
 
+		<h3>Video trim</h3>
+		<p>Can set video playback start and end point in seconds.</p>
+
+		<h4>Set trim</h4>
+		<p>If you trim an unsaved auto recorded video then that video will be saved.</p>
```

Comment:
"Please note that unsaved auto-recorded videos will become saved if you trim them."